.env.example

```
@@ -4,6 +4,8 @@ ETH_ADDRESS=
 ETH_PRIVKEY=
 ETH_MNEMONIC=
 ETH_RPC_URL=https://rpc.brovider.xyz/5
+STARKNET_MAINNET_RPC_URL=https://starknet-mainnet.infura.io/v3/46a5dd9727bf48d4a132672d3f376146
+STARKNET_GOERLI_RPC_URL=https://starknet-goerli.infura.io/v3/46a5dd9727bf48d4a132672d3f376146
```

Better to not expose our API key here, I've added them on DigitalOcean
```suggestion
STARKNET_MAINNET_RPC_URL=
STARKNET_GOERLI_RPC_URL=
```

---

Actually nevermind, it doesn't seem to use our paid account.

---

This API key is from checkpoint-template.